internal/resources/metal/connection/datasource_schema.go

```diff
@@ -50,7 +50,7 @@ func dataSourceSchema(ctx context.Context) schema.Schema {
 				Computed:    true,
 			},
 			"speed": schema.StringAttribute{
-				Description: fmt.Sprintf("Port speed. Possible values are %s", allowedSpeedsString()),
+				Description: fmt.Sprintf("Port speed. Required for a_side connections. Values will be in the format '<number>Mbps' or '<number>Gpbs'"),
```

Comment:
The .md should reflect the change. Perhaps we could give some examples speeds in the description.

Reply:
I updated the docs, but there was already some drift between the description in docs and the description in code; we probably should pick one and use it in both places, but which one to pick is an exercise left to the reviewer.

Reply:
For comparison:
- [description in resource docs](https://github.com/equinix/terraform-provider-equinix/pull/610/files#diff-6ed6746523e1ea4d3f4fb0b43c3e840c082c07d57be197e2a4bbc269b311ad84R148)
- [description in resource code](https://github.com/equinix/terraform-provider-equinix/pull/610/files#diff-cf57b498cae31d4a345adc8449c2308c16e2e5570703a97e49122c71f43e112cR97)